dev-libs/quazip/quazip-0.7.6.recipe

```diff
@@ -56,7 +57,7 @@ BUILD()
 
 	cmake .. \
 		$cmakeDirArgs \
-		-DCMAKE_ROOT=$libDir/cmake
+		-DINSTALL_PATH=$prefix/lib/cmake
```

Comment:
I've take a look at the [cmake module][0]. It seems that this module utilizes `pkg-config` to do it's work, so I'd say it's architecture independent and can be put in `$dataDir/cmake`.

[0]: https://github.com/stachenov/quazip/blob/d5eaa8be998da2430ffd43e0eefa617cf1550ca3/FindQuaZip.cmake